[Chromium]  Assertion errors while loading URLs by typing on the URL bar

I get this many times when typing an URL (or some search terms) in the URL bar

```
12:49:39.948  W  java.lang.AssertionError: Must only be used on a single thread.
12:49:39.948  W  	at org.chromium.base.ThreadUtils$ThreadChecker.assertOnValidThread(ThreadUtils.java:72)
12:49:39.948  W  	at org.chromium.base.ObserverList.iterator(ObserverList.java:145)
12:49:39.948  W  	at org.chromium.content.browser.webcontents.WebContentsObserverProxy.didStartLoading(WebContentsObserverProxy.java:164)
12:49:39.948  W  	at J.N.MAqmDh4t(Native Method)
12:49:39.948  W  	at org.jni_zero.GEN_JNI.org_chromium_content_browser_framehost_NavigationControllerImpl_loadUrl(GEN_JNI.java:1800)
12:49:39.948  W  	at org.chromium.content.browser.framehost.NavigationControllerImplJni.loadUrl(NavigationControllerImplJni.java:156)
12:49:39.948  W  	at org.chromium.content.browser.framehost.NavigationControllerImpl.loadUrl(NavigationControllerImpl.java:178)
12:49:39.948  W  	at org.chromium.wolvic.Tab.loadUrl(Tab.java:81)
12:49:39.948  W  	at com.igalia.wolvic.browser.api.impl.SessionImpl.loadUri(SessionImpl.java:80)
12:49:39.948  W  	at com.igalia.wolvic.browser.engine.Session.loadUri(Session.java:877)
12:49:39.948  W  	at com.igalia.wolvic.browser.engine.Session.loadUri(Session.java:867)
12:49:39.948  W  	at com.igalia.wolvic.ui.views.NavigationURLBar.handleURLEdit(NavigationURLBar.java:384)
12:49:39.948  W  	at com.igalia.wolvic.ui.views.NavigationURLBar.lambda$initialize$1$com-igalia-wolvic-ui-views-NavigationURLBar(NavigationURLBar.java:160)
12:49:39.948  W  	at com.igalia.wolvic.ui.views.NavigationURLBar$$ExternalSyntheticLambda4.onEditorAction(Unknown Source:2)
12:49:39.948  W  	at android.widget.TextView.onEditorAction(TextView.java:7010)
12:49:39.948  W  	at com.android.internal.widget.EditableInputConnection.performEditorAction(EditableInputConnection.java:138)
12:49:39.948  W  	at android.view.inputmethod.InputConnectionWrapper.performEditorAction(InputConnectionWrapper.java:205)
12:49:39.948  W  	at android.view.inputmethod.InputConnectionWrapper.performEditorAction(InputConnectionWrapper.java:205)
12:49:39.948  W  	at com.igalia.wolvic.ui.widgets.KeyboardWidget.lambda$handleDone$17(KeyboardWidget.java:1008)
12:49:39.948  W  	at com.igalia.wolvic.ui.widgets.KeyboardWidget$$ExternalSyntheticLambda2.run(Unknown Source:4)
12:49:39.949  W  	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1663)
12:49:39.949  W  	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1655)
12:49:39.949  W  	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:285)
12:49:39.949  W  	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1155)
12:49:39.949  W  	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1993)
12:49:39.949  W  	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1941)
12:49:39.949  W  	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

--- a/app/src/common/shared/com/igalia/wolvic/ui/views/NavigationURLBar.java
+++ b/app/src/common/shared/com/igalia/wolvic/ui/views/NavigationURLBar.java
@@ -6,6 +6,7 @@
 package com.igalia.wolvic.ui.views;
 
 import android.annotation.SuppressLint;
+import android.app.Activity;
 import android.content.ClipData;
 import android.content.ClipboardManager;
 import android.content.Context;
@@ -157,7 +158,9 @@ public class NavigationURLBar extends FrameLayout {
         mBinding.urlEditText.setOnEditorActionListener((aTextView, actionId, event) -> {
             if (actionId == EditorInfo.IME_ACTION_DONE || actionId == EditorInfo.IME_ACTION_SEARCH
                     || actionId == EditorInfo.IME_ACTION_GO || actionId == EditorInfo.IME_ACTION_SEND) {
-                handleURLEdit(aTextView.getText().toString());
+                ((Activity)getContext()).runOnUiThread(() -> {
+                    handleURLEdit(aTextView.getText().toString());
+                });
                 return true;
             }
             return false;

